Less logs

yalh76 opened the pull request:
## Problem

- *file log is huge*

## Solution

- *Applying official log configuration from https://matrix-org.github.io/synapse/latest/usage/configuration/logging_sample_config.html*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

yalh76 commented:
!testme

Josue-T commented:
Hello thank you for the pull request, did you check that fail2ban still work after the change ?

yalh76 commented:
> Hello thank you for the pull request, did you check that fail2ban still work after the change ?

no, I didn't check the fail2ban, but now it's check and fixed in https://github.com/YunoHost-Apps/synapse_ynh/pull/506/commits/f5db77eef5b1a2f6f9d01fdb37f5fd81758b8a6c

The main difference, I introduced is on  :
`format: '%(asctime)s - %(name)s - %(lineno)d - %(levelname)s - %(request)s- %(message)s'`

new value is 
`format: '%(asctime)s - %(name)s - %(lineno)d - %(levelname)s - %(request)s - %(message)s'`

the difference is the the space added between `%(request)s` and `- %(message)s`

yalh76 commented:
!testme

yalh76 commented:
!testme

yalh76 commented:
!testme

Josue-T reviewed (approved):
Thank you for the improvement and checking fail2ban.
